fusor-ember-cli/app/components/radio-button.js

```diff
@@ -0,0 +1,2 @@
+import RadioButton from 'ember-radio-button/components/radio-button';
+export default RadioButton;
```

Comment:
This is basically pulling in the code for {{radio-button}} since this is the name of component. We are re-opening the same class, not creating a new class and extending it.